### ISSUE TITLE
Reverse Approval of Confirmation Mode ?

### ISSUE THREAD

**Issue by LittleDuke** [SergeyMosin/Appointments#222]
Has this already been considered?

This request would change the "confirmation" from having the requestor click back to confirm to having the calendar owner confirm the slot within some period of time or it would expire.

1. User requests time slot
2. Calendar owner confirms or declines request, or 
3. Slot becomes available again if no action is taken within some configurable time period

Thoughts?

-dvd

**Comment by JimKillock** [comment]
Rather han “flipping,” this needs two confirmations; one from the requestor (to confirm email details); second from the calendar owner. so
# User requests time slot
# User conforms time slot and email address
# Calendar owner confirms or declines request, or
# Slot becomes available again if no action is taken within some configurable time period

**Comment by LittleDuke** [comment]
I agree that there is value in having the user request be confirmed by the user so that you can verify their email address -- in fact if someone does not confirm it within some adjustable period of time then it simple gets deleted.
